### PR TITLE
update collab 2

### PR DESCRIPTION
**Pull request by phyliciachristel**
- prompts other user when one disconnects
- prevents unauthorised access by other users
- removed parameters from the link
- configured matching with complexity + language
- syntax highlighting !!

other stuff:
- does not persist on refresh, didn't really handle this so the connection/disconnection on refresh is abit weird still
- toast might show disconnect + connect many times when joining a room cos of react strict mode

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "@emotion/react": "^11.11.1",
     "@emotion/styled": "^11.11.0",
+    "@monaco-editor/react": "^4.6.0",
```
Comment (cfyjoanna) [inline frontend/package.json:8]: if you're not using the old one can you help do npm uninstall react-monaco-editor (or just remove it from package json) <3

File: frontend/src/pages/collab.js
Comment (cfyjoanna) [inline frontend/src/pages/collab.js:1]: I cannot find it but remove the ugly disconnect button pls :)

Comment (cfyjoanna) [inline frontend/src/pages/collab.js:1]: I'm not sure where it is but can you capitalize it to JavaScript <333
image

**Review by cfyjoanna — COMMENTED** [review]
lgtm i j have some nitpicks 🍞😍😍

File: frontend/package.json (hunk 1 above)
Comment (phyliciachristel) [reply frontend/package.json:8]: i need the old one as a dependency for this to work <3

File: frontend/src/pages/collab.js
Comment (phyliciachristel) [reply frontend/src/pages/collab.js:1]: </3

**Review by cfyjoanna — APPROVED** [review]
lgtm 😋😋

**Pull request merged**